src/test/java/api/loans/anonymization/AnonymizeLoansImmediatelyAPITests.java

```diff
@@ -4,18 +4,16 @@
 import static org.hamcrest.Matchers.not;
 import static org.hamcrest.junit.MatcherAssert.assertThat;
 
-import java.net.MalformedURLException;
 import java.util.UUID;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.TimeoutException;
 
 import api.support.builders.CheckOutByBarcodeRequestBuilder;
 import api.support.builders.LoanHistoryConfigurationBuilder;
-import org.folio.circulation.support.http.client.IndividualResource;
 import org.joda.time.DateTime;
-import org.joda.time.DateTimeUtils;
+import org.joda.time.DateTimeZone;
 import org.junit.Test;
 
+import org.folio.circulation.support.http.client.IndividualResource;
```

Comment:
Why is this separate from the other `org.x` imports?

Reply:
That was the "old" formatting. Changed.